ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -161,60 +175,56 @@ def check_volumedrivers(logger):
                     if vp.guid in VolumedriverHealthCheck.MACHINE_DETAILS.vpools_guids:
```

Comment:
Same here, put the ```else``` stack here and return from the function and make the entire ```if``` stack unindented once